### ISSUE TITLE
Resetting the form clears the list of choices

### ISSUE THREAD

**Issue by mscart** [Choices-js/Choices#1053]
I have a form with some filters that make an ajax request to display a table. When I want to reset the filters ( $("#filter-form").trigger("reset");) all the select elements created with choices.js are empty and I have nothing left to select. I have to refresh the page to repopulate the select forms.
any ideea ?

**Comment by masiorama** [comment]
I'm experiencing the same issue... how can we handle this scenario?

**Comment by kumardeepakxyz** [comment]
Having the same issue. Is there a fix for this?

**Comment by Sogl** [comment]
The same issue when I clear form inputs. Still no fix?

**Comment by jamesperrin** [comment]
Ran into the same problem.

My band-aid is to add a click addEventListener() for the reset button. 

Then, use Choices method [removeActiveItems()](https://github.com/Choices-js/Choices#removeactiveitemsexcludedid "Choices removeActiveItems() information") to clear the select elements.

This also means you'll need to add code to reset all the other form fields.

```js
document.addEventListener('DOMContentLoaded', function () {
	const element = document.querySelector('.js-choice');

	const choicejsOptions = {
		allowHTML: false,
		removeItemButton: true,
		searchEnabled: true,
		searchChoices: true,
		searchFields: ['label', 'value'],
		placeholder: true,
	};

	const services = new Choices('.js-choice__Services', choicejsOptions);

	document.querySelector('input[type="reset"]').addEventListener('click', function (evt) {
		evt.preventDefault;

		services.removeActiveItems();
	});
});
```

**Comment by ceciiiron** [comment]
Even the documentation shows the bug. It removes all of the options when I click the reset button

**Comment by bpieres** [comment]
No news? I can't clean my selections... Thanks

**Comment by MatheusRich** [comment]
A work-around (snippet from a Stimulus controller):

```js
const choices = new Choices(this.element, {
  allowHTML: false,
  removeItemButton: this.element.multiple,
});

const form = this.element.closest("form");
if (!form) return;

form.addEventListener("reset", () => {
  choices.destroy();
  choices.init();
});```

**Comment by Xon** [comment]
Implemented as part of #1166
